_includes/comp-man/checklist.md

```diff
@@ -17,17 +17,23 @@ Magento requires three cron jobs, all running as the [Magento file system owner]
 
 To verify your cron jobs are set up properly, enter the following command as a user with `root` privileges:
 
-	crontab -u <magento file system owner> -l
+```bash
+crontab -u <magento file system owner> -l
+```
 
 For example, if your Magento file system owner is named `magento_user`, enter:
 
-	crontab -u magento_user -l
+```bash
+crontab -u magento_user -l
+```
 
 Results similar to the following should display:
 
-	* * * * * /usr/bin/php /var/www/magento2/bin/magento cron:run | grep -v "Ran jobs by schedule" >> /var/www/magento2/var/log/magento.cron.log
-	* * * * * /usr/bin/php /var/www/magento2/update/cron.php >> /var/www/magento2/var/log/update.cron.log
-	* * * * * /usr/bin/php /var/www/magento2/bin/magento setup:cron:run >> /var/www/magento2/var/log/setup.cron.log
+```terminal
```

Comment:
`bash` commands you enter + `terminal` output?

Reply:
Yes